src/ci/docker/host-x86_64/dist-x86_64-linux/build-clang.sh

```diff
@@ -39,6 +39,7 @@ hide_output \
       -DLLVM_INCLUDE_TESTS=OFF \
       -DLLVM_INCLUDE_EXAMPLES=OFF \
       -DLLVM_ENABLE_PROJECTS="clang;lld;compiler-rt;bolt" \
+      -DLLVM_BINUTILS_INCDIR="/rustroot/lib/gcc/x86_64-pc-linux-gnu/9.5.0/plugin/include/" \
```

Comment:
This should have link to build-gcc.sh with gcc version?

Reply:
Yes I guess it makes more sense to lift a GCC_VERSION variable into the Dockerfile and use that for both